functorch/_src/aot_autograd.py

```diff
@@ -1795,27 +1796,30 @@ def functional_call(*args, **kwargs):
         num_params_buffers=params_len,
     )
 
-    compiled_fn = None
+    full_args = []
+    full_args.extend(params_flat)
+    full_args.extend(args)
 
-    @wraps(functional_call)
-    def compiled_f(*args):
-        nonlocal compiled_fn
-        if compiled_fn is None:
-            compiled_fn = create_aot_dispatcher_function(
-                functional_call,
-                args,
-                aot_config,
-            )
-        return compiled_fn(args)
-
-    def forward(*args):
-        return compiled_f(
-            *params_flat,
-            *args,
-        )
+    compiled_fn = create_aot_dispatcher_function(
```

Comment:
can you underscore `create_aot_dispatcher_function` like in the source PR?

Reply:
You're welcome to underscore functions in a follow up PR, there's a lot more identifiers than `create_aot_dispatcher_function` that need underscoring, and underscoring the function here contributed unnecessary line noise to the PR

Reply:
In general, the functorch convention is that all identifiers in `functorch/_src` are considered private (even if the identifier doesn't have a leading underscore), and public identifiers are selectively reexported in `functorch` itself). So it is unnecessary to mark identifiers private on a one-by-one basis.